src/ui/components/Timeline/index.tsx

```diff
@@ -125,15 +125,18 @@ class Timeline extends Component<PropsFromRedux> {
       setTimelineState,
       isTrimming,
     } = this.props;
-    const hoveringOverMarker = !!hoveredComment;
+    // if the user clicked on a comment marker, we already seek to the comment's
+    // execution point, so we don't want to seek a second time here
+    const clickedOnCommentMarker =
```

Comment:
I'm very glad that we're not using the redux state anymore, because that *does* seem super brittle, but I'm a little confused as to why this click handler isn't *on* the element that we're checking for? Wouldn't a more elegant way of doing this same check be to set the onClickHandler on the comment markers themselves?

Reply:
We have an onClick handler on the comment markers themselves (which does almost the same thing), but we also need this handler when the user doesn't click on a comment marker and we want to prevent that both handlers run for the same click.